scripts/clatd.systemd

```diff
@@ -12,6 +12,8 @@
 Description=464XLAT CLAT daemon
 Documentation=man:clatd(8)
 After=network-online.target
+Before=nss-lookup.target
```

Comment:
Shouldn't this be `After=`?

Reply:
@jivanpal Would appreciate it if you could explain why you are using `Before=` here. It seems to me that `After=` would be the appropriate dependency to use - considering the PR is titled «ensure service starts ***after*** DNS queries can be resolved».